action.yml

```diff
@@ -22,9 +47,7 @@ runs:
           // The pattern for JIRA ticket format
           const jiraPattern = /FIR-\d+/g
           if (!jiraPattern.test(prTitle)) {
```

Comment:
As discussed, let's add some placeholder for no ticket present

Reply:
Another thing I'm thinking, could it also be DOC-... or something else. Should we add a more generic ticket pattern

Reply:
I see an official jira pattern here, but it can be an overhead: https://confluence.atlassian.com/bitbucketserverkb/integrating-with-custom-jira-issue-key-in-bitbucket-server-779171880.html

Reply:
My only problem are the false-positives, e.g. `feat: this fixes ERROR-110` will succeed since we have a "ticket" name in the title. But this is a very small edge case.